data/korath/korath missions.txt

```diff
@@ -577,3 +577,19 @@ mission "Efreti: Distress Call 8"
 			`	You return to your ship and try plugging the chip into your systems again, which now displays two files: one in the Korath language, and one in your own. You open the file in your language, and it reads: "The <EDC-ship> is a ship of grief, bringing great pain in memories that sing for generations. Love is a void in the souls of many. Know all that <first> <last> is a true Efret by heart, if not by skin. The <EDC-ship> is theirs, to save lives and keep memories."`
 			`	You carefully store the chip in a safe place.`
 
+
+mission "Exile: Boarding Worldship Korath"
+	boarding
+	minor
+	destination "Earth"
+	source
+		government "Korath" "Korath (Civilian)"
+		category "Superheavy"
+		outfits "Afterburner (Stellar Class)"
```

Comment:
For the rest of your text to make sense, it may be better to have a minimum crew requirement of, say, 10000. I'm not sure if that's possible in the ship filters.

There is another superheavy Korath ship, and it is conceivable that it could be equipped with Stellar-class afterburners.

Another way to differentiate the two Korath superheavies is to add

```julia
not outfits "Microbot Grinder"
```

The Ikatila'ej is the only ship with a spinal mount, so no others could have that outfit. You'd never have in a city, for fear that it may consume the populace.

Reply:
Definitely the microbot grinder is the better sorting factor, even if it may presently be unnecessary.

```suggestion
		category "Superheavy"
		not outfits "Microbot Grinder"
```

Reply:
Yeah I was looking for unique things on the A'awoj, not the Ikatila'ej, so I missed that, thank you!